portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,48 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Eyad Alsilimy</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <style> h1 {text-align: center;} </style>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Welcome to My Portfolio</h1>
+      <p>&emsp;&emsp;I spent the first part of my STEP internship working on it. I am a rising junior in Electrical Engineering with a focus in Embedded Systems. I hope to provide as much value to
+        the world as I can. I think I am going to do this through technology. I am Egyptian with
+        Canadian citizenship, living in America.I love a lot of things, especially when they involve others.
+        I end spending my free time watching youtube about various things (educational and interesting),
+        playing video games (LoL, CSGO, RotMG), and hanging out with friends. I love talking about music and
+        am currently avidly listen to artist like Playboi Carti, Giveon, Travis Scott, and Gunna.</p>
```

Comment:
Use 4 spaces of indention of wrapping lines. You can check: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping